src/OpenLoco/src/Vehicles/VehicleHead.cpp

```diff
@@ -2552,6 +2552,11 @@ namespace OpenLoco::Vehicles
 
     uint8_t VehicleHead::getLoadingModifier(const VehicleBogie* bogie)
     {
+        if (Config::get().disableVehicleLoadPenaltyCheat)
+        {
+            return 1;
+        }
+
```

Comment:
L2598 is logic that should probably still occur. I origianlly wanted to make this a nice pure function when i yanked it out of where it was being used but i didn't want to duplicate the logic for L2598.

Reply:
oh yeah good catch, fixed